### PR TITLE
[API] reduce customer update serialization

### PR DESCRIPTION
**Pull request by SirDomin**
| Q               | A
| --------------- | -----
| Branch?         | master
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no/
| Related tickets | 
| License         | MIT

### CODE REVIEW

File: src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/Customer.xml [hunk 1]
```diff
@@ -71,9 +71,12 @@
             <itemOperation name="shop_put">
                 <attribute name="method">PUT</attribute>
                 <attribute name="path">/shop/customers/{id}</attribute>
-                <attribute name="denormalization_context">
+                <attribute name="normalization_context">
                     <attribute name="groups">shop:customer:update</attribute>
                 </attribute>
+                <attribute name="denormalization_context">
+                    <attribute name="groups">shop:customer:denormalize</attribute>
+                </attribute>
```
Comment (GSadee) [inline src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/Customer.xml:79]:
```suggestion
                <attribute name="denormalization_context">
                    <attribute name="groups">shop:customer:update</attribute>
                </attribute>
                <attribute name="normalization_context">
                    <attribute name="groups">shop:customer:read</attribute>
                </attribute>
```

File: tests/Api/Shop/CustomerUpdateTest.php [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Tests\Api\Shop;
+
+use Sylius\Tests\Api\JsonApiTestCase;
+use Symfony\Component\HttpFoundation\Response;
+
+class CustomerUpdateTest extends JsonApiTestCase
```
Comment (Zales0123) [inline tests/Api/Shop/CustomerUpdateTest.php:17]: `final` 💃

Comment (Zales0123) [inline tests/Api/Shop/CustomerUpdateTest.php:17]: Also, to keep the convention used in this catalog, it should probably be named `CustomersTest`

**Review by GSadee — APPROVED** [review]

**Review by Zales0123 — APPROVED** [review]

**Comment by GSadee** [comment]
Thanks, @SirDomin! :1st_place_medal:

**Pull request merged**